infra/charts/management-service/README.md

```diff
@@ -0,0 +1,99 @@
+# xp-management
+
+---
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square)
+![AppVersion: 0.0.1](https://img.shields.io/badge/AppVersion-0.0.1-informational?style=flat-square)
+
+A Helm chart for Kubernetes Deployment of the XP Managment Service
+
+## Introduction
+
+This Helm chart installs [Management Service](https://github.com/gojek/xp/management-service) and all its dependencies in a Kubernetes cluster.
+
+## Prerequisites
+
+To use the charts here, [Helm](https://helm.sh/) must be configured for your
+Kubernetes cluster. Setting up Kubernetes and Helm is outside the scope of
+this README. Please refer to the Kubernetes and Helm documentation.
+
+- **Helm 3.0+** – This chart was tested with Helm v3.7.1, but it is also expected to work with earlier Helm versions
+- **Kubernetes 1.18+** – This chart was tested with GKE v1.20.x, but it's possible it works with earlier k8s versions too.
+
+## Installation
+
+### Add Helm repository
+
+```sh
+$ helm repo add xp https://turing-ml.github.io/charts
+```
+
+### Installing the chart
+
+This command will install XP Management Service release named `management-service` in the `default` namespace.
+Default chart values will be used for the installation:
+```shell
+$ helm install xp xp/management-service
+```
+
+You can (and most likely, should) override the default configuration with values suitable for your installation.
+Refer to [Configuration](#configuration) section for the detailed description of available configuration keys.
+
+You can also refer to [values.minimal.yaml](./values.minimal.yaml) to check a minimal configuration that needs
+to be provided for XP Management Service installation.
+
+### Uninstalling the chart
+
+To uninstall `management-service` release:
+```shell
+$ helm uninstall management-service
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release,
+except for postgresql PVC, those will have to be removed manually.
+
+## Configuration
+
+The following table lists the configurable parameters of the XP Management Service chart and their default values.
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.mlp.encryption.key | string | `nil` | Global MLP Encryption Key to be used by all MLP components |
```

Comment:
We don't have any global values in the chart now, right?

Reply:
We declare some helper templates which uses it - https://github.com/gojek/turing-experiments/blob/add-helm-chart-build-publish/infra/charts/management-service/templates/_helpers.tpl#L76.